WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/breakpoint/BreakpointType.java

```diff
@@ -20,11 +20,11 @@
 	@SerializedName("me")
 	METHOD,
 	
```

Comment:
Isn't this more about the scope of the break point? Would be helpful to perhaps rename the class that way.